ruby-monday-blog/spec/models/post_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe Post do
```

Comment:
I thought you meant after the  describe not before.  Didn't realize you can leave that out.  I removed that now.